src/lib/snack-bar/snack-bar-container.ts

```diff
@@ -42,8 +42,8 @@ export type SnackBarState = 'initial' | 'visible' | 'complete' | 'void';
 
 // TODO(jelbourn): we can't use constants from animation.ts here because you can't use
 // a text interpolation in anything that is analyzed statically with ngc (for AoT compile).
-export const SHOW_ANIMATION = '225ms cubic-bezier(0.4,0.0,1,1)';
-export const HIDE_ANIMATION = '195ms cubic-bezier(0.0,0.0,0.2,1)';
+export const SHOW_ANIMATION = '375ms cubic-bezier(0.4,0.0,0.2,1)';
```

Comment:
As you noted, we are not using the correct easing curves for our animations.  However it appears we just have the curves reversed.

From looking at the spec once again:
[Entering the screen](https://material.io/guidelines/motion/movement.html#movement-movement-in-out-of-screen-bounds) should use the [deceleration curve](https://material.io/guidelines/motion/duration-easing.html#duration-easing-natural-easing-curves) - `cubic-bezier(0.0, 0.0, 0.2, 1)`
[Permanently leaving the screen](https://material.io/guidelines/motion/movement.html#movement-movement-in-out-of-screen-bounds) should use the [acceleration curve](https://material.io/guidelines/motion/duration-easing.html#duration-easing-natural-easing-curves)  - `cubic-bezier(0.4, 0.0, 1, 1)` 


The durations appear to be correct as they are non-complex transitions.  These are also guided by the spec found [here](https://material.io/guidelines/motion/duration-easing.html#duration-easing-common-durations).